AwesomeCalculator/app/src/main/java/com/example/drcreeper/awesomecalculator/asynktasks/DeleteHistoryListAsyncTask.java

```diff
@@ -0,0 +1,35 @@
+package com.example.drcreeper.awesomecalculator.asynktasks;
+
+import android.content.Context;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteOpenHelper;
+import android.os.AsyncTask;
+
+import com.example.drcreeper.awesomecalculator.historywriter.HistoryDatabaseOpenHelper;
+import com.example.drcreeper.awesomecalculator.historywriter.HistoryDatabaseScheme;
+
+public class DeleteHistoryListAsyncTask extends AsyncTask<Void,Void,Void> {
+
+    Context context;
```

Comment:
Fields must be private by default unless there is real need to make them more visible. The same for methods.